pkgs/by-name/su/surrealdb/package.nix

```
@@ -5,29 +5,23 @@
   fetchFromGitHub,
   pkg-config,
   openssl,
-  rocksdb_8_3,
+  rocksdb,
   testers,
   surrealdb,
   darwin,
   protobuf,
-}:
-
-let
-  rocksdb = rocksdb_8_3;
-in
-rustPlatform.buildRustPackage rec {
+}: rustPlatform.buildRustPackage rec {
   pname = "surrealdb";
-  version = "2.0.2";
+  version = "2.2.0";
 
   src = fetchFromGitHub {
     owner = "surrealdb";
     repo = "surrealdb";
     rev = "v${version}";
-    hash = "sha256-kTTZx/IXXJrkC0qm4Nx0hYPbricNjwFshCq0aFYCTo0=";
+    hash = "sha256-gsIeoxSfbHHSdpPn6xAB/t5w3cLtpu6MjTuf5xsI6wI=";
   };
 
-  useFetchCargoVendor = true;
-  cargoHash = "sha256-yJfpxklHD/qbEoUS4OMAzlc9/0d/iW3EABU6AK/Maqo=";
+  cargoHash = "sha256-OXcQsDCiT3seMQhyKEKfC8pcd4MXwbql5+ZDGGkhPMI=";
```

```suggestion
	useFetchCargoVendor = true;
  cargoHash = "sha256-OXcQsDCiT3seMQhyKEKfC8pcd4MXwbql5+ZDGGkhPMI=";
```

This'll also update the `cargoHash`.